### DISCUSSION TITLE
Dificuldade de inserção na área | Mercado de trabalho em Porto Alegre

### DISCUSSION THREAD

**Discussion by valkervieira** [frontendbr/forum#1407] [Discussão]
Pessoal, recentemente fiz um desabafo na #1397 mas lá não era o local então vou falar aqui. Já li todos os posts de carreira do fórum mas gostaria de alguma dica mais direta ou só uma palavra de apoio.

Eu trabalho atualmente como instrutor/professor em uma escola de cursos profissionalizantes, apesar de eu gostar bastante do meu trabalho eu tenho pouco espaço para crescimento, além de gostar bastante da área de dev e meio que sentir que "eu nasci pra isso". Acontece que eu não consigo me inserir no mercado de jeito nenhum.

O principal motivo da minha frustração é que eu vejo gente de tudo quanto é lado, desde amigos a inclusive alunos meus conseguindo emprego na área sabendo MUITO menos do que eu sei, quando eu pergunto pra essas pessoas como elas conseguiram emprego na área eles respondem a mesma coisa: "mandei currículo, fui na entrevista, me chamaram", todos sem portfólio, sem nenhum trabalho pra mostrar, usando tecnologias super antigas, tem aluno meu que pegou vaga de front sem nunca ter ouvido falar de JS e morrer de medo de abrir um terminal. Eu fico feliz de ver um aluno meu bem empregado, mas é muito frustrante saber que ele ganha bem mais que eu trabalhando numa área que eu adoraria trabalhar e não sabendo 1/3 do que eu sei. Isso ta acabando com a minha auto-estima e com o meu psicológico.

Basicamente eu sinto que eu estou estudando em vão, hoje eu domino HTML5 e tenho cuidado com código semântico, to até pensando em me arriscar num PUG da vida, domino CSS inclusive media queries, keyframes, flexbox, grid e curto bastante usar Sass. Manjo bem de JS e ES6+. Meu bundler favorito é o Parcel. Pensando no mercado de trabalho nos últimos tempos tenho aprendido React e me sinto bem confortável com a ferramenta. Leio livros, manjo de inglês, acompanho as tendências, mas quando chega na entrevista eu nunca chego na parte técnica. Sempre sou dispensado antes. Esses dias inclusive fui dispensado em um processo numa empresa que tinha o site com erros amadores ridículos como o mobile com o width bem maior que o viewport, um formulario dentro de um iframe com barra de rolagem, refiz o layout em poucas horas e mandei pra eles tudo corrigido e bem mais otimizado pra ver se me davam uma moral e nem me responderam.

Lendo na #911 as características que os entrevistadores pedem são praticamente uma descrição sumária da minha personalidade, eu sou super comunicativo e interessado, sempre pergunto e me projeto, busco sempre estar ligado no que há de mais novo no mercado, mas parece que não serve de nada.

Eu sei que a dica principal é fazer projetos por conta própria mas eu não tenho muito tempo já estando empregado em outra área, durante o dia trabalho e durante a noite preciso cuidar da minha filha, ainda assim tento ao máximo fazer um projetinho aqui e ali.

Se não bastasse tudo isso, aqui em Porto Alegre eu praticamente não acho vaga fora das plataformas tradicionais como infojobs da vida, que pra mim nunca serviram de nada. Alguém tem uma dica, está passando ou já passou por isso? Sinto que minha cabeça vai explodir de frustração nesse ritmo... Parece que é tão fácil pra tanta gente mas pra mim é impossível sabe...

**Comment by MrS4w** [comment]
Acho que você deveria procurar por vagas remotas ou até mesmo mudar de cidade (se possível) eu por exemplo, sou do interior de Pernambuco, onde as vagas aparecem mais em Recife e raramente em Caruaru (principalmente vagas de Júnior), por mais que eu soubesse as tecnologias, não valeria a pena pra nenhuma empresa ou até eu mesmo me deslocar todo dia pra trabalhar, além de ser caro e não ter diversidades de meios de transporte, graças á uns fatores atualmente eu moro em São Paulo e o que falta aqui são desenvolvedores, em mais ou menos um mês mandando e-mail pra todas as vagas possíveis, fui chamado pra algumas entrevistas, recebi um "não" em quase todas, em uma em específico acabei fazendo networking com o @felipefialho que em plena entrevista me deu umas dicas legais e que logo em mais ou menos uns 13 dias eu consegui um emprego pra trabalhar com tecnologias que nunca tinha usado, atualmente estou bem contente porquê aqui dentro me deram espaço para aprender de fato e hoje já estou agregando valor junto com o resto do pessoal :smile:

**Comment by mikaelhadler** [comment]
Uma dica é utilizar o LinkedIn, ao menos ao meu ver aqui em porto alegre é uma das plataformas mais utilizadas para contratação dev. Aqui no próprio fórum existe a divulgação de vagas para porto alegre, onde por exemplo, vi a vaga que tinha disponível na empresa em que trabalho atualmente.

Falando nisto, temos vagas. Se quiser posso lhe indicar para o RH se for ajudar.

Atualmente tem vaga para front aqui em porto alegre :)

**Comment by valkervieira** [comment]
> Acho que você deveria procurar por vagas remotas ou até mesmo mudar de cidade (se possível) eu por exemplo, sou do interior de Pernambuco, onde as vagas aparecem mais em Recife e raramente em Caruaru (principalmente vagas de Júnior), por mais que eu soubesse as tecnologias, não valeria a pena pra nenhuma empresa ou até eu mesmo me deslocar todo dia pra trabalhar, além de ser caro e não ter diversidades de meios de transporte, graças á uns fatores atualmente eu moro em São Paulo e o que falta aqui são desenvolvedores, em mais ou menos um mês mandando e-mail pra todas as vagas possíveis, fui chamado pra algumas entrevistas, recebi um "não" em quase todas, em uma em específico acabei fazendo networking com o @felipefialho que em plena entrevista me deu umas dicas legais e que logo em mais ou menos uns 13 dias eu consegui um emprego pra trabalhar com tecnologias que nunca tinha usado, atualmente estou bem contente porquê aqui dentro me deram espaço para aprender de fato e hoje já estou agregando valor junto com o resto do pessoal

Eu estive pensando nisso, nas minhas férias estive em Florianópolis e amei a cidade, mandei alguns currículos e fiz algumas entrevistas, o que eu achei legal é que em todas me deram feedback, e inclusive as que não me chamaram para a entrevista me responderam, mas depois de uma conversa com a minha esposa eu acabei tirando a ideia da cabeça, visto que eu e ela estudamos na universidade federal aqui em Porto Alegre e ela ta em um curso bem complicado de passar, além de termos uma filha, nossa rede de apoio está toda por aqui. Mas eu tenho muita vontade de mudar de cidade sim, eu mesmo sou paulista, se não fosse esses fatores eu já estaria me candidatando pra vagas por lá.

> Uma dica é utilizar o LinkedIn, ao menos ao meu ver aqui em porto alegre é uma das plataformas mais utilizadas para contratação dev. Aqui no próprio fórum existe a divulgação de vagas para porto alegre, onde por exemplo, vi a vaga que tinha disponível na empresa em que trabalho atualmente.

Pois confesso que o Linkedin nunca foi a plataforma que eu dei mais foco, mas agora que você citou vou dar uma recauchutada no meu perfil e dar um pouquinho mais de atenção, eu era bem ativo quando trabalhei na área do suporte, pensei que era a área de mais foco por aqui.

> Falando nisto, temos vagas. Se quiser posso lhe indicar para o RH se for ajudar.

Poxa, adoraria! Se quiser falar algo internamente meu currículo ta disponível aqui https://github.com/borghettinho/cv ou podes me passar o contato se preferir. Muito obrigado!

**Comment by mikaelhadler** [comment]
@borghettinho Contato de indicação feito brother, boa sorte = )

**Comment by kazzkiq** [comment]
Você tá buscando vagas em qual "categoria"? Júnior? Pleno? Sênior?

Pela minha experiência dentro de empresas, o que pode estar acontecendo é você buscar uma vaga onde o RH/entrevistadores esperam mais experiência prática na área, com maior rodagem por outras empresas, e como você trabalha dando aula, isso pode deixar eles com um pé atrás.

Geralmente esse requisito (experiência prévia em outras empresas) vai pesando mais conforme o nível que a vaga exige, ou seja: Uma vaga Júnior (ou de estágio) tende a não dar tanta importância pra quem não tem um vasto histórico empresarial. Já vagas buscando pessoas mais experientes certamente vão colocar um peso bem maior nisso. Digo isso porque já vi candidatos serem escolhidos por terem mais experiência de mercado que outros, sendo que muitas vezes os dois tiveram performances parecidas nas entrevistas.

Se esse for realmente seu caso, eu recomendo duas coisas (você pode escolher uma delas, ou aplicar as duas):

1. Buscar vagas nível Júnior, inicialmente. Pensa nisso como um investimento, depois de conseguir entrar em uma empresa, você pode tentar crescer dentro dela (o que não vai ser difícil uma vez que você já possui conhecimento técnico na área), ou eventualmente buscar outra vaga melhor já tendo uma outra experiência empresarial no seu currículo.

2. Arrumar tempo de alguma forma e começar a criar projetos open-source aqui no GitHub. Essa é uma boa forma de demonstrar seu trabalho prático, mesmo sem ter referências de outras empresas.

**Comment by valkervieira** [comment]
> Buscar vagas nível Júnior, inicialmente. Pensa nisso como um investimento, depois de conseguir entrar em uma empresa, você pode tentar crescer dentro dela (o que não vai ser difícil uma vez que você já possui conhecimento técnico na área), ou eventualmente buscar outra vaga melhor já tendo uma outra experiência empresarial no seu currículo.

Eu costumo procurar de nível Jr., pra ser sincero talvez eu até seja demasiado enfático nisso, já que normalmente quando a descrição da vaga pede "sólido conhecimento/experiência" em alguma stack eu já não me arrisco pois imagino que a empresa espera um profissional bem auto-suficiente, o que não é meu caso ainda, já que eu dependendo muito de ler documentação pra resolver problemas.

> Arrumar tempo de alguma forma e começar a criar projetos open-source aqui no GitHub. Essa é uma boa forma de demonstrar seu trabalho prático, mesmo sem ter referências de outras empresas.

É justamente o que eu mais to patinando pra fazer mas que eu acredito que me traga maior retorno, além disso eu já me coloco a disposição pra fazer freela para conhecidos e indicações, mas as oportunidades ainda não surgiram. E em sites especializados eu acredito que a falta de um portfolio tem me prejudicado.
